src/test/resources/application-test.yml

```diff
@@ -2,7 +2,7 @@ workspace:
   buffer:
     pool-id: workspace_manager_v1
     instance-url: https://buffer.tools.integ.envs.broadinstitute.org
-    buffer-client-credential-file-path: rendered/buffer-client-sa-account.json
+    client-credential-file-path: rendered/buffer-client-sa-account.json
```

Comment:
Is there reflection somewhere that maps this name into a java object?

Reply:
This will be read using spring's `ConfigurationProperties` annotation (https://github.com/DataBiosphere/terra-workspace-manager/blob/a9cd0addddadce40fd75b0311dfe93eaecb8bd57/src/main/java/bio/terra/workspace/app/configuration/external/BufferServiceConfiguration.java#L15) which turns the path into a String